server/src/models/user.js

```diff
@@ -1,37 +1,36 @@
-'use strict';
-const Sequelize = require('sequelize')
+const Sequelize = require('sequelize');
 
 module.exports = (sequelize, DataTypes) => {
   var Model = sequelize.define('User', {
-    id: {
-      type: DataTypes.UUID(),
-      primaryKey: true,
-      defaultValue: Sequelize.UUIDV4,
-      allowNull: false,
-    },
-    name: {
-      type: DataTypes.STRING,
-      allowNull: false,
+      id: {
+        type: DataTypes.UUID(),
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4,
```

Comment:
Now that I look at it, this seems like a mistake. It's weird to mix ``Sequelize`` with ``DataTypes``

Reply:
`type: DataTypes.UUID,
defaultValue: DataTypes.UUIDV1` probably? http://docs.sequelizejs.com/variable/index.html

Reply:
Sounds good